docs/recipes/react.md

```diff
@@ -10,19 +7,23 @@ Translations: [Español](https://github.com/avajs/ava-docs/blob/master/es_ES/doc
 
 The first step is setting up Babel to transpile JSX code from the tests by adding an AVA section to your package.json:
 
+```console
+$ npm install --save-dev @babel/preset-react
+```
```

Comment:
It's odd to have this section here. There's no instructions on `@babel/register` either. Perhaps just leave it out, so it's like it was before?